src/ansys/geometry/core/connection/product_instance.py

```diff
@@ -298,6 +303,22 @@ def get_available_port():
     return port
 
 
+def _wait_for_backend(host: str, port: int, timeout: int):
```

Comment:
This is though for local installations right? Maybe we could default the host to `localhost` as in other functions of this module. Also, I think having a default timeout would be nice.

Reply:
Its a private function, default parameters are coming from upstream